silver/models/payments.py

```diff
@@ -154,6 +155,10 @@ def _log_unsuccessful_transition(self, transition_name):
         })
 
     @property
+    def transactions(self):
+        return Transaction.objects.filter(payment=self).all()
```

Comment:
You could use self.transaction_set.all() and avoid importing the Transaction class at all, as it might create some circular dependencies.